packages/legacy/core/App/components/misc/NewQRView.tsx

```diff
@@ -39,7 +39,7 @@ const NewQRView: React.FC<Props> = ({ defaultToConnect, handleCodeScan, error, e
   const { width } = useWindowDimensions()
   const qrSize = width - 40
   const [store] = useStore()
-  const { showScanHelp, showScanButton } = useConfiguration()
+  const [{ showScanHelp, showScanButton }] = useServices([TOKENS.CONFIG])
```

Comment:
This destructuring syntax seems a little odd. Can it be simplified to just be `{}` or `[]` rather than both?